skeleton/ci/gitops-template/githubactions/.github/workflows/gitops-promotion.yml

```diff
@@ -15,28 +15,30 @@ env:
   IMAGE_REGISTRY: ${{ secrets.IMAGE_REGISTRY }}
 
   # Used to verify the image signature and attestation
-  COSIGN_PUBLIC_KEY: ${{ secrets.COSIGN_PUBLIC_KEY }}
+  COSIGN_PUBLIC_KEY: ${{ vars.COSIGN_PUBLIC_KEY }}
```

Comment:
Should the `COSIGN_PUBLIC_KEY` be visible? Perhaps there is a reason I'm not knowledgeable with for this.